src/presentational-components/styled-components/table.js

```diff
@@ -28,11 +28,10 @@ Tbody.propTypes = {
   children: childrenPropType.isRequired
 };
 
-export const TableCell = styled(({ shrink, children, ...props }) => (
```

Comment:
This file can stay as is, not sending the shrink parameter in the order-item would be enough.